Sources/Base/OAuth2ClientConfig.swift

```diff
@@ -139,6 +145,11 @@ open class OAuth2ClientConfig {
 		if let assume = settings["token_assume_unexpired"] as? Bool {
 			accessTokenAssumeUnexpired = assume
 		}
+		
+		if let usePKCE = settings["use_PKCE"] as? Bool {
```

Comment:
For consistency use `use_pkce`? Also please document this where all the other settings are documented.

Reply:
Done.